sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -554,6 +554,9 @@ def add(self, message):
         :rtype: None
         :raises: :class: ~azure.servicebus.exceptions.MessageContentTooLarge, when exceeding the size limit.
         """
+        if isinstance(message, dict):
+            temp_message = Message(message.pop('body'), **message)
```

Comment:
Nit: Don't need the body pop (here and elsewhere), the ** expansion will do the right thing even for positional args, and you'll get a proper typeerror if they're not present. (instead of a keyerror which would be nonstandard)

```suggestion
            temp_message = Message(**message)
```